app/views/internal/classified_listings/index.html.erb

```diff
@@ -11,7 +11,12 @@
         </div>
         <div class="form-group">
           <%= label_tag(:filter, "Category") %>
-          <%= select_tag(:filter, options_for_select([""] + ClassifiedListing.categories_available.keys, params[:filter])) %>
+          <%= select_tag(:filter, options_for_select(ClassifiedListing.categories_available.keys, params[:filter]), include_blank: true) %>
+        </div>
+        <div class="form-group">
+          <%= label_tag(:published, "Published only") %>
+          <%= hidden_field_tag(:published, 0) %>
+          <%= check_box_tag(:published, 1, true) %>
```

Comment:
I second what Anna has flagged re: `Published only` checkbox bug; I captured my observations in the video below - hope it's helpful! 🙏 

https://www.loom.com/share/198b5444f1ab4465afbd1461e1fcb2cf

Reply:
1. What the checkbox is supposed to do is to toggle between all listings (including published "no") and only the ones with a published status of "yes". It should never only show unpublished listings. Maybe the label of the checkbox is not great, so I'm open to suggestions here  😃 

2. The checkbox behavior was somewhat intentional, as this is an internal tool with one main user (@michael-tharrington) and this is going to save him one click. Given that it confused two people though I'll change it to behave more like a traditional checkbox.

BTW, @lightalloy was only referencing point 2. above in her comment, not point 1.

Reply:
ahhhhh ok I understand. Yes the checkbox label didnt make it very clear to me. Thanks for explaining.